src/components/GraphiQL.js

```diff
@@ -64,6 +65,12 @@ export class GraphiQL extends React.Component {
     editorTheme: PropTypes.string,
     onToggleHistory: PropTypes.func,
     ResultsTooltip: PropTypes.any,
+    plugins: PropTypes.arrayOf(
+      PropTypes.shape({
+        title: PropTypes.string.isRequired,
```

Comment:
* `title` is what will be displayed as the tab title in the `PluginsPane`. 
* `plugin` is a component with the render prop that passes in a function that takes the result string as an argument and returns whatever jsx it wants to be rendered.